dbt/include/spark/macros/adapters.sql

```diff
@@ -229,9 +229,29 @@
   {% endfor %}
 {% endmacro %}
 
+{% macro get_column_comment_sql(column_name, column_dict) -%}
+  {% if column_name in column_dict and column_dict[column_name]["description"] -%}
+    {% set column_comment_clause = "comment '" ~ column_dict[column_name]["description"] ~ "'" %}
```

Comment:
Filter with 
```
| replace("'", "\\'")
```

Reply:
is this what you mean? could TLDR's the filter's purpose for me?

```suggestion
    {% set column_comment_clause = "comment '" ~ column_dict[column_name]["description"] ~ "'" | replace("'", "\\'")%}
```

Reply:
Its to escape single-quotes to allow them to be used in the comments.

Reply:
you need it to operate only on the description, I think here you have it operating either on the whole string or on the final ' (not the best at Jinja).  In my local copy I moved the extraction and filtering to separate 'set' statement.

Reply:
```jinja2
{% set escaped_description = column_dict[column_name]["description"] | replace("'", "\\'") %}
{% set column_comment_clause = "comment '" ~ escaped_description ~ "'" %}
```